assert/object_match.ts

```diff
@@ -3,7 +3,7 @@
 import { assertEquals } from "./equals.ts";
 
 /**
- * Make an assertion that `actual` object is a subset of `expected` object,
+ * Make an assertion that `actual` object is a superset of `expected` object,
```

Comment:
`actual` needs to include all of `expected` fields. I think `actual` is called `superset` in this case.

Reply:
I think it'd be a little easier to understand if we say that `expected` is a subset of `actual`.